sdk/formrecognizer/azure-ai-formrecognizer/azure/ai/formrecognizer/_generated/_operations_mixin.py

```diff
@@ -42,9 +42,9 @@ def begin_analyze_business_card_async(
 
         :param include_text_details: Include text lines and element references in the result.
         :type include_text_details: bool
-        :param locale: Locale of the business card. Supported locales include: en-AU, en-CA, en-GB, en-
-         IN, en-US(default).
-        :type locale: str
+        :param locale: Locale of the input document. Supported locales include: en-AU, en-CA, en-GB,
+         en-IN, en-US(default).
```

Comment:
So invoice is only `en-US` should the docstring be updated to reflect that?

Reply:
This is generated code, it is updated in the exposed docs

Reply:
changos. sorry about that!